pkg/jx/cmd/controller_pipelinerunner.go

```diff
@@ -203,9 +203,9 @@ func (o *ControllerPipelineRunnerOptions) startPipelineRun(w http.ResponseWriter
 
 	pr := &StepCreateTaskOptions{}
 	if pj.Type == prowapi.PostsubmitJob {
-		pr.PipelineKind = jenkinsfile.PipelineKindRelease
+		pr.PipelineKind = v1alpha1.PipelineKindRelease
```

Comment:
I would use a named import everywhere instead of `v1alpha1`. It is more readable. For instance:

```Go
import (
jxsyntax "github.com/jenkins-x/jx/pkg/syntax/syntax.jenkins.io/v1alpha1"
)

Reply:
Yeah, that's a good suggestion. Will do!